2 failing tests in pytest

Hey, while working on some other issues I found the following errors in pytest

```
===================================== FAILURES ======================================
__________________________ test_export_to_dict__from_list ___________________________

alias_list = [Alias(name='alix-test-echo', command='alix test working!', description='alix test shortcut', tags=['a', 'b'], created...me.datetime(2025, 10, 24, 16, 34, 21, 653023), used_count=0, shell=None, last_used=None, usage_history=[], group=None)]
porter_data = {'aliases': [{'command': 'alix test working!', 'created_at': '2025-10-24T16:34:21.653023', 'description': 'alix test s...23', 'description': 'alix test shortcut', 'last_used': None, ...}], 'count': 2, 'exported_at': <ANY>, 'version': '1.0'}

    def test_export_to_dict__from_list(alias_list, porter_data):
        porter = AliasPorter()
        export = porter.export_to_dict(alias_list)
    
>       assert export == porter_data
E       AssertionError: assert {'aliases': [...rsion': '1.0'} == {'aliases': [...rsion': '1.0'}
E         
E         Omitting 3 identical items, use -vv to show
E         Differing items:
E         {'aliases': [{'command': 'alix test working!', 'created_at': '2025-10-24T16:34:21.653023', 'description': 'alix test s... test working!', 'created_at
E         
E         ...Full output truncated (2 lines hidden), use '-vv' to show

tests/test_porter.py:12: AssertionError
_________________________ test_export_to_dict__from_storage _________________________

mock_list_all = <MagicMock name='list_all' id='140047008544528'>
alias_list = [Alias(name='alix-test-echo', command='alix test working!', description='alix test shortcut', tags=['a', 'b'], created...me.datetime(2025, 10, 24, 16, 34, 21, 653023), used_count=0, shell=None, last_used=None, usage_history=[], group=None)]
porter_data = {'aliases': [{'command': 'alix test working!', 'created_at': '2025-10-24T16:34:21.653023', 'description': 'alix test s...23', 'description': 'alix test shortcut', 'last_used': None, ...}], 'count': 2, 'exported_at': <ANY>, 'version': '1.0'}

    @patch.object(AliasStorage, "list_all")
    def test_export_to_dict__from_storage(mock_list_all, alias_list, porter_data):
        mock_list_all.return_value = alias_list
    
        porter = AliasPorter()
        export = porter.export_to_dict()
    
>       assert export == porter_data
E       AssertionError: assert {'aliases': [...rsion': '1.0'} == {'aliases': [...rsion': '1.0'}
E         
E         Omitting 3 identical items, use -vv to show
E         Differing items:
E         {'aliases': [{'command': 'alix test working!', 'created_at': '2025-10-24T16:34:21.653023', 'description': 'alix test s... test working!', 'created_at
E         
E         ...Full output truncated (2 lines hidden), use '-vv' to show

tests/test_porter.py:22: AssertionError
```

Need to investigate and resolve this.

--- a/tests/conftest.py
+++ b/tests/conftest.py
@@ -51,6 +51,7 @@ def porter_data() -> Dict[str, Any]:
                 "used_count": 0,
                 "last_used": None,
                 "usage_history": [],
+                "group": None,
             },
             {
                 "name": "alix-test-echo",
@@ -62,6 +63,7 @@ def porter_data() -> Dict[str, Any]:
                 "used_count": 0,
                 "last_used": None,
                 "usage_history": [],
+                "group": None,
             },
         ],
     }

